src/examples/notifications/alerts/AlertType.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Alert, Title, Close } from '@zendeskgarden/react-notifications';
+
+const StyledSpacer = styled.div`
+  height: ${p => p.theme.space.md};
+`;
+
+const Example = () => (
+  <>
+    <Alert type="info">
+      <Title>Info</Title>
+      Watering plants in a garden will cause them to grow.
+      <Close aria-label="Close Alert" />
```

Comment:
Going to change the titles, because the examples should demonstrate what an Info title would look/feel like:

<title>Partly cloudy today</title>
Sun-loving plants might need some extra TLC